STREAMING_INGESTION.md

```diff
@@ -0,0 +1,32 @@
+# Streaming Ingestion
+
+## Configurations
+
+We still use the same configurations, but the `host` and the `path` configurations need to be changed so that they will point to our Ingestion API.
+
+The `host` configuration will have the following values, depending on which environment you want to ingest data.
+
+- AWS Asia Pacific (Seoul)
+  - **ap02.records.in.treasuredata.com**
+
+- AWS Asia Pacific (Tokyo):
+  - **ap01.records.in.treasuredata.com**
+
+- AWS East
+  - **us01.records.in.treasuredata.com**
+
+- AWS EU
+  - **eu01.records.in.treasuredata.com**
+
+The `path` configuration should be always `/`
```

Comment:
Can we just remove the option then?

Reply:
The property should be `pathname`, as it is mentioned in the configuration section of the documentation. We want to explicitly set the `pathname` to `/`